### PR TITLE
KVM direct downloads

### PR DESCRIPTION
**Pull request by nvazquez**
Add documentation for direct downloads on KVM

### CODE REVIEW

**Comment by PaulAngus** [comment]
requesting docbuild

**Comment by nvazquez** [comment]
Ping @rhtyd @borisstoyanov @PaulAngus @anuragaw @shwstppr can you please review?

File: source/adminguide/templates/_bypass-secondary-storage-kvm.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+Certificates for direct downloads synchronization task
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Certificates are uploaded to the running hosts in a zone at a certain moment. However, the number of running hosts may change and new hosts added to the zone may not include the certificate uploaded to the rest of the hosts.
+
+CloudStack provides a way to synchronize certificates across all the running hosts on each zone. The global setting 'direct.download.certificate.background.task.interval' defines the interval in which the synchronization task will run. This task will:
+
+- Iterate through each enabled zone
+- Get the running hosts in a zone
+- Check which hosts need the certificates which have been already uploaded to other hosts
```
Comment (PaulAngus) [inline source/adminguide/templates/_bypass-secondary-storage-kvm.rst]:
```suggestion
- Check which hosts are missing the certificates which have been already uploaded to other hosts
```

File: source/adminguide/templates/_bypass-secondary-storage-kvm.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+   -  **Direct Download**. It will be shown in the UI when KVM is selected as hypervisor. Choose Yes for enabling the bypassing secondary storage option.
+
+   -  **Checksum**. Optional field. If this field is populated, the checksum is compared to the downloaded template checksum when the template is downloaded to primary storage at deployment time.
+
+After the template is registered, it is automatically available for VM deployments.
+
+Uploading certificates for direct downloads
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For HTTPS direct downloads, the KVM hosts on a zone should need certificates.
```
[the remark this answers is not in the document]
Comment (nvazquez) [reply source/adminguide/templates/_bypass-secondary-storage-kvm.rst]: Fixed

**Comment by nvazquez** [comment]
Thanks @PaulAngus

**Pull request merged**